lib/plugins/sass.js

```diff
@@ -0,0 +1,31 @@
+import { join } from 'path';
+import pathParse from 'path-parse';
+import { CSS_IMPORT } from '../../packages/helper-grammar-regex-collection/index.js';
+import insertLink from '../insert-link';
+import preset from '../pattern-preset';
+import relativeFile from '../resolver/relative-file.js';
+
+export default class Sass {
+
+  static resolve({ path, target }) {
+    const { dir, base, ext } = pathParse(target);
+    const prefixedTarget = join(dir, `_${base}`);
+
+    return [
+      relativeFile({ path, target: `${prefixedTarget}.scss` }),
```

Comment:
Good point. I removed `${prefixedTarget}.${ext}` in favour of the additional file extension `.sass`